CodeGeneration/Sources/generate-swiftsyntax/MiscFile.swift

```diff
@@ -0,0 +1,114 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+import SwiftSyntaxBuilder
+import SyntaxSupport
+import Utils
+
+let miscFile = SourceFile {
+  ExtensionDecl("""
+    \(generateCopyrightHeader(for: "generate-swiftsyntax"))
+    extension Syntax
+    """) {
+    VariableDecl(
+      modifiers: [DeclModifierSyntax(name: Token.public), DeclModifier(name: Token.static)],
+      name: IdentifierPattern("structure"),
+      type: TypeAnnotation(
+        colon: .colon,
+        type: SimpleTypeIdentifier("SyntaxNodeStructure")
+      )
+    ) {
+      
+      ReturnStmt(expression: FunctionCallExpr(
+        calledExpression: MemberAccessExpr(".choices"),
```

Comment:
Can you use the `callee` overload here so you don't need to specify `leftParen`?